### PR TITLE
added graalpy

### PR DESCRIPTION
**Pull request by tomasstupka**

### CODE REVIEW

File: starter-core/src/main/java/io/micronaut/starter/feature/graallanguages/MicronautGraalPyFeature.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        return Category.API;
+    }
+
+    @Override
+    public void apply(GeneratorContext generatorContext) {
+        generatorContext.addDependency(MICRONAUT_GRAALPY_DEPENDENCY);
+
+        if (generatorContext.getBuildTool() == BuildTool.MAVEN) {
+            BuildProperties buildProperties = generatorContext.getBuildProperties();
+            // XXX GP is this the right place to declare the version?
```
Comment (timfel) [inline starter-core/src/main/java/io/micronaut/starter/feature/graallanguages/MicronautGraalPyFeature.java]: A question for @graemerocher, right?

File: starter-core/src/main/java/io/micronaut/starter/feature/graallanguages/MicronautGraalPyFeature.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    @Override
+    public String getTitle() {
+        return "Micronaut GraalPy Extension";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Provides GraalPy integration with Micronaut";
```
Comment (fniephaus) [inline starter-core/src/main/java/io/micronaut/starter/feature/graallanguages/MicronautGraalPyFeature.java]: I think it'd be good if this mentions "Python", for example:
```suggestion
        return "Adds support for Python using GraalPy";
```

**Review by sdelamo — CHANGES_REQUESTED** [review]
@tomasstupka you have to sign the CLA.

File: starter-core/src/main/java/io/micronaut/starter/feature/graallanguages/MicronautGraalPyFeature.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    @Override
+    public String getThirdPartyDocumentation() {
+        return "https://graalvm.org/python";
+    }
+
+    @Override
+    public String getMicronautDocumentation() {
+        return "https://micronaut-projects.github.io/micronaut-graalpy/latest/guide/";
```
Comment (sdelamo) [inline starter-core/src/main/java/io/micronaut/starter/feature/graallanguages/MicronautGraalPyFeature.java]:
```suggestion
        return "https://micronaut-projects.github.io/micronaut-graal-languages/latest/guide/";
```

**Review by sdelamo — APPROVED** [review]
I have done small changes. I have renamed the feature to `graalpy` and fix the link the module docs.

**Comment by steve-s** [comment]
```
Error: R] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 61.30 s <<< FAILURE! -- in example.micronaut.FooTest
Error: R] example.micronaut.FooTest -- Time elapsed: 61.30 s <<< ERROR!
  io.micronaut.context.exceptions.BeanInstantiationException: 
  Bean definition [org.hibernate.SessionFactory] could not be loaded: Error instantiating bean of type  [org.hibernate.boot.SessionFactoryBuilder]
   
  Message: Test resources service wasn't able to revolve expression 'jpa.default.properties.hibernate.connection.url': java.net.http.HttpTimeoutException: request timed out
  Path Taken: SessionFactoryPerJpaConfigurationFactory.buildHibernateSessionFactoryBuilder(SessionFactoryBuilder sessionFactoryBuilder) --> SessionFactoryPerJpaConfigurationFactory.buildHibernateSessionFactoryBuilder([SessionFactoryBuilder sessionFactoryBuilder]) --> SessionFactoryPerJpaConfigurationFactory.buildHibernateSessionFactoryBuilder(Metadata metadata,[JpaConfiguration jpaConfiguration])
  	at io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1999)
  	at io.micronaut.context.DefaultApplicationContext.initializeContext(DefaultApplicationContext.java:303)
```

looks like some connectivity issue, maybe we can try restarting the job?

**Comment by steve-s** [comment]
@sdelamo any thoughts on the failed job? Can you try starting the jobs again? I thought that pushing artificial change could trigger them, but it didn't.

**Pull request merged**